homeassistant/components/nad/media_player.py

```diff
@@ -200,12 +200,16 @@ def update(self):
         self._volume = self.calc_volume(self._nad_receiver.main_volume("?"))
         self._source = self._source_dict.get(self._nad_receiver.main_source("?"))
 
-    def calc_volume(self, decibel):
+    def calc_volume(self, decibel: int) -> float:
```

Comment:
Return should be `Optional[float]` since you may return `None`.

You can import `Optional` from `typing`

Reply:
Ah, I should have seen that myself. Thanks :+1: